marlowe-cli/marlowe-cli.cabal

```diff
@@ -107,10 +108,12 @@ library
     , errors
     , extra
     , marlowe-cardano
+    , megaparsec
```

Comment:
I think that `attoparsec` performs better, but that's unlikely to matter for contract files.